Bottom: 2ef10257086e9abb92eae056cc170b244ccbd7cf
Top:    981a355a6bdd6695eea7613c929d82915ee403f6
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 00:53:26 +0000

Remove left over debug


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 27e7c54..9f4b0e0 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1252,8 +1252,6 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
 #endif
                                 copper_appearance,
                                 NULL));
-
-      //break;
     }
